proto/elys/margin/query.proto

```diff
@@ -20,7 +21,7 @@ service Query {
   }
   
   // Queries a list of GetMTP items.
-  rpc GetMTP (MTPRequest) returns (MTPResponse) {
+  rpc QueryGetMTP (MTPRequest) returns (MTPResponse) {
```

Comment:
I think the GRPC query QueryGetMTP is not a good naming - `MTP` seems to be better

Reply:
@jelysn Okay I got it.

Reply:
@jelysn Done